tests/data/missing_component_version/SPDXJSONExample-v2.3.spdx.json

```diff
@@ -101,7 +101,7 @@
     "licenseDeclared" : "(LGPL-2.0-only AND LicenseRef-3)",
     "licenseInfoFromFiles" : [ "GPL-2.0-only", "LicenseRef-2", "LicenseRef-1" ],
     "name" : "glibc",
-    "originator" : "Organization: ExampleCodeInspect (contact@example.com)",
+    "originator" : "NOASSERTION",
```

Comment:
These changes exemplify the test suite changes I made over and over. I removed the originator value, changing it to "NOASSERTION" and did provide a value for supplier.